lib/tasks/phases.rake

```diff
@@ -0,0 +1,10 @@
+namespace :phases do
+  desc "Sets the final date of the phase to the end of the day"
+  task ends_on_the_end_of_the_day: :environment do |_, args|
```

Comment:
You are not using the block arguments, so you don't need to define them

`:environment do`